Avoid un-needed objects.inv files drom d'loading for docs build
diff -ur Flask-0.9.orig/docs/conf.py Flask-0.9/docs/conf.py
--- docs/conf.py	2012-07-01 19:10:19.000000000 +0800
+++ docs/conf.py	2013-06-09 00:21:57.757485401 +0800
@@ -243,14 +243,6 @@
 # The depth of the table of contents in toc.ncx.
 #epub_tocdepth = 3
 
-intersphinx_mapping = {
-    'http://docs.python.org/dev': None,
-    'http://werkzeug.pocoo.org/docs/': None,
-    'http://www.sqlalchemy.org/docs/': None,
-    'http://wtforms.simplecodes.com/docs/0.5/': None,
-    'http://discorporate.us/projects/Blinker/docs/1.1/': None
-}
-
 pygments_style = 'flask_theme_support.FlaskyStyle'
 
 # fall back if theme is not there
